### DISCUSSION TITLE
能像Magisk那样随机包名把LSPosed隐藏了么？

### DISCUSSION THREAD

**Discussion by StrugglingAndroid** [LSPosed/LSPosed#1489] [Q&A] [answered]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

能像Magisk那样随机包名把LSPosed隐藏了么？ 

### Describe the solution you'd like/描述你想要的解决方案

能像Magisk那样随机包名把LSPosed隐藏了么？

**Comment by yujincheng08** [comment, marked as answer]
不能。

**Comment by StrugglingAndroid** [comment]
回答简洁有力、、

**Comment by JasonKhew96** [comment]
> 回答简洁有力、、

@AfterFinal 没必要，都寄生了，扫不出包名

**Comment by StrugglingAndroid** [comment]
ok，3Q

**Comment by canyie** [comment]
#238
